### PR TITLE
transaction execution docs update

### PR DESCRIPTION
**Pull request by janezpodhostnik**
## Description

Added a short description of transaction execution. 
______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/concepts/transaction-lifecycle.md [hunk 1]
```diff
@@ -39,13 +39,79 @@ Collection Nodes pass the transaction collections to the Consensus Nodes for the
 
 ## Execution
 
-_FVM Documentation coming soon..._
+Once the transaction reaches the execution node as part of a collection it is validated and processed by the Flow Virtual Machine (FVM) following steps illustrated by the image below.
 
-### Fee Payment
-In the execution process, the relevant fees are deducted from the account identified as the `payer` in the transacction. Flow's flexible transaction structure allows accounts other than the user to pay transaction fees. For more info on [transaction structure](https://github.com/onflow/flow/blob/master/docs/content/concepts/accounts-and-keys.md#anatomy-of-a-transaction) and [transaction signing](https://developers.flow.com/learn/concepts/accounts-and-keys). 
+![Transaction Execution](fvm-transaction-execuiton.png)
```
Comment (franklywatson) [inline docs/content/concepts/transaction-lifecycle.md:44]:
```suggestion
![Transaction Execution](fvm-transaction-execution.png)
```

**Review by franklywatson — COMMENTED** [review]
A general comment is that we mention errors being thrown in several stages of this and I know I have also seen many people ask 'what does this error mean' in discord. So I am wondering whether we should have some summary of the error messages that will be possible here to help make things completely clear and available to search engines..

**Comment by janezpodhostnik** [comment]
I think that is going to be added here: https://github.com/onflow/flow-go-sdk/pull/344 but once it is, This doc could link to it.

**Review by zhangchiqing — APPROVED** [review]

File: docs/content/concepts/transaction-lifecycle.md [hunk 2]
```diff
@@ -39,13 +39,79 @@ Collection Nodes pass the transaction collections to the Consensus Nodes for the
...
+During this step the users transaction code is executed. In case an error occurs the transaction continues with the [Execution error step](#execution-error-step).
+
+#### Stop metering execution effort
+
+Now that the users part of the transaction is executed, metering is stopped. The metered quantities will be used for fee deduction later.
+In case any metering limits were hit during execution, the execution would continue with the [Execution error step](#execution-error-step).
+
+#### Check accounts' storage limits
+
+In this step the all accounts whose storage was changed during the transaction execution will be checked that their [storage capacity](/concepts/storage#storage-capacity-of-the-payer) does not exceed their [storage used](/concepts/storage#storage-used). If it does, an error is produced and the transaction continues with the [Execution error step](#execution-error-step).
```
Comment (j1010001) [inline docs/content/concepts/transaction-lifecycle.md:94]:
```suggestion
In this step, all accounts whose storage was changed during the transaction execution, will be checked to ensure their [storage capacity](/concepts/storage#storage-capacity-of-the-payer) does not exceed their [storage used](/concepts/storage#storage-used). If it does, an error is produced and the transaction continues with the [Execution error step](#execution-error-step).
```

**Review by j1010001 — APPROVED** [review]

**Pull request merged**